src/main/java/seedu/duke/shooter/Shooter.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.shooter;
+
+public class Shooter {
+    public String name;
+    private int power;
+    private int skill;
+
+    public Shooter(String name) {
+        this.name = name;
+    }
+    public void printSelfInfo(){
+        System.out.println("--------Player Info--------");
+        System.out.println("Name: "+this.name);
+        printPower();
+        printSkill();
+        System.out.println("---------------------------");
+    }
+
+    protected void printPower(){
+
+    }
+
+    protected void printSkill(){
+        System.out.print("Skill:");
+        for (int i=0; i<3;i++){
+            if (i<skill){
+                System.out.print(" 000");
+            }else{
+                System.out.print(" OOO");
+            }
```

Comment:
the printed characters don't make any sense, please change it.

Reply:
You are right, it is clear in IntelliJ, but I failed to consider different systems.
![image](https://github.com/AY2324S2-CS2113-F15-3/tp/assets/86831302/76183971-5f5f-488b-a149-edb998a35edc)

Reply:
I don't think you understood what I was saying. I mean why print "000" and "OOO"? What is that supposed to mean?